### PR TITLE
Discontinued GIVfarm cards

### PR DESCRIPTION
**Pull request by mateodaza**
Sets discontinuation for production farms to stop showing on September 8th as it is exactly 3 weeks after August 18th.

Before that date we'll display them by default and tell the user to remove all liquidity from there

@laurenluz  @divine-comedian we can change the date if you disagree

Test with production env

### CODE REVIEW

**Comment by mateodaza** [comment]
100% GIV staking on mainnet
GIV / DAI uniswap on mainnet
GIV/ETH sushiswap on gnosis chain
GIV / xDAI honeyswap on gnosis chain

This is the message
image

File: src/components/homeTabs/GIVfarm.tsx
Comment (MohammadPCh) [inline src/components/homeTabs/GIVfarm.tsx:166]: please move GIVstaking configs to the out of the component, I think the networkProvider.ts is a good place for it.

File: src/types/config.ts
Comment (MohammadPCh) [inline src/types/config.ts:7]: may we should decide between this and the active property in the simpleStakingConfig,
they are same!

**Review by MohammadPCh — CHANGES_REQUESTED** [review]
@mateodaza Thanks,
I suggest so  improvements also I have written 2 more suggestions here:

- I think we don't need to add complexity to our app if we want to archive some farms at a specific time, we can easily change them to archived at that time.
- please add configs in development configs too. it helps testers to test it easier.

thanks.

**Comment by mateodaza** [comment]
As soon as we merge this, I'll create a new PR for manually archiving in Sept 8th

**Review by MohammadPCh — APPROVED** [review]
Thanks @mateodaza

**Pull request merged**
